File: Modul-3/Soal-1/PRAK301-2410817320001-NazlaSalsabila.py
a, b, c = map(int, input("Masukkan tiga angka: ").split())

if a > b:
    a, b = b, a
if a > c:
    a, c = c, a
if b > c:
    b, c = c, b

print(f"Hasil dari pengurutan dari nilai terkecil ke terbesar: {a} {b} {c}")

File: Modul-3/Soal-1/PRAK301-2410817320001-NazlaSalsabila.c
#include <stdio.h>

int main() {
    int a, b, c, temp;

    printf("Masukkan tiga angka: ");
    scanf("%d %d %d", &a, &b, &c);

    if (a > b) {
        temp = a; a = b; b = temp;
    }
    if (a > c) {
        temp = a; a = c; c = temp;
    }
    if (b > c) {
        temp = b; b = c; c = temp;
    }

    printf("Hasil dari pengurutan dari nilai terkecil ke terbesar: %d %d %d\n", a, b, c);
    
    return 0;
}